app/code/Magento/Inventory/Model/OptionSource/SourceItemStatus.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model\OptionSource;
+
+use Magento\Framework\Data\OptionSourceInterface;
+
+/**
+ * Class SourceItemStatus
+ *
+ * @api
+ */
+class SourceItemStatus implements OptionSourceInterface
+{
+    /**#@+
+     * Source items status values
+     */
+    const SOURCE_ITEM_STATUS_OUT_OF_STOCK = 0;
+    const SOURCE_ITEM_STATUS_IN_STOCK = 1;
```

Comment:
> why should we introduce new constants if we already have these constants declared in SourceItemInterface ?

There are NOT new constants. Constants has been moved so Source model (like as in another source models)